src/OpenTK/Platform/X11/XI2MouseKeyboard.cs

```diff
@@ -167,11 +167,32 @@ internal static bool IsSupported(IntPtr display)
             return false;
         }
 
+        private bool GetIndicatorState(string name)
+        {
+            bool state = false;
+            bool lockState = false;
+            int ndx = 0;
+            IntPtr map = new IntPtr ();
+            bool real = false;
+        
+            IntPtr atom = Functions.XInternAtom(API.DefaultDisplay, name, true);
+            Xkb.GetNamedIndicator(API.DefaultDisplay, atom, out ndx, out state, out map, out real);
+            
+            return state;
+        }
+
         KeyboardState IKeyboardDriver2.GetState()
         {
             lock (Sync)
             {
                 KeyboardState state = new KeyboardState();
+
+                if (Xkb.IsSupported (API.DefaultDisplay))
```

Comment:
Do we need to call IsSupported? Is ok to mix xinput2 and xkb? Speaking of which doesn't X11Keyboard.cs also need updating with something like this?